inc/api.php

```
@@ -405,9 +438,93 @@ function djz_get_gamipress_user_data($request) {
+        'status' => ['completed', 'processing'],
+        'type' => 'shop_order',
+    ];
+
+    $orders = wc_get_orders($args);
+    $count = 0;
+    $target_slugs = ['events', 'tickets', 'congressos', 'workshops', 'social', 'festivais', 'pass'];
+
+    if ($orders) {
+        foreach ($orders as $order) {
+            foreach ($order->get_items() as $item) {
+                $product_id = $item->get_product_id();
```

PHPMD (2.15.0)

[error] The variable $product_id is not named in camelCase. (undefined)

(CamelCaseVariableName)